cmd/minishift/cmd/config/util.go

```diff
@@ -70,3 +71,10 @@ func SetBool(m MinishiftConfig, name string, val string) error {
 	m[name] = b
 	return nil
 }
+
+func SetSlice(m MinishiftConfig, name string, val string) error {
+	if val != "" {
```

Comment:
👍 

I think we discussed we need to document this somewhere. I would think somewhere in the persistent config section.

I am also wondering whether one could not write some standalone unit tests for this SetXYZ.